llama_stack/providers/adapters/memory/weaviate/__init__.py

```diff
@@ -0,0 +1,8 @@
+from llama_stack.distribution.datatypes import RemoteProviderConfig
+
+async def get_adapter_impl(config: RemoteProviderConfig, _deps):
+    from .weaviate import WeaviateMemoryAdapter
+
+    impl = WeaviateMemoryAdapter(config.url, config.username, config.password)
```

Comment:
the config does not define url / username / password but instead only defines a collection?

Reply:
the cluster url and api_key for weaviate will come in via the WeaviateRequestProviderData class, in this case should I just get rid of the WeaviateConfig class?

Reply:
done